src/sas/sascalc/dataloader/readers/IgorReader.py

```diff
@@ -113,14 +113,14 @@ def read(self, filename=None):
                 # Get total bin number
                 
             i_tot_row += 1
-        i_tot_row = math.ceil(math.sqrt(i_tot_row)) - 1
-        #print "i_tot", i_tot_row
+        i_tot_row = int(math.ceil(math.sqrt(i_tot_row)) - 1)
+        # print "i_tot", i_tot_row

```

Comment:
I agree, the math in the original can lead to inaccuracies. I'm suggesting the calculation become:
`
i_tot_row = int(math.sqrt(i_tot_row) + 0.5)
`
We use this convention because int always truncates the input value. Any floating point value between 4.5 and 5.499999, after adding 0.5, will become integer 5. In the case you gave, the square root of 16384 will be a float of 128, but as you probably know, floating point values aren't exactly the desired value. They also can be slightly greater or less than the desired value. Adding 0.5 will ensure the end value is always above 128 so the expected value will be the result.